### DISCUSSION TITLE
Can't get past application stage glitch

### DISCUSSION THREAD

**Discussion by LucasMontoya4** [community/community#123107] [GitHub Education]
### Hi everyone,

I originally applied for the education student developer pack, and the image of my photo ID was rejected (It wasn't such a good photo to be fair). So I have done everything I was asked to do and tried to reapply, and at the stage where you upload a photo of your ID, I upload a much better photo, and when I click 'process my application', the page just refreshes as if I had done nothing.

Has anyone else had this problem?

**Comment by davevad93** [comment]
Hi @LucasMontoya4 ,

I personally haven't experienced that issue so I'm not sure what might be the cause, try this:

1. Switch browser and check if the problem persists.
2. If the above solution doesn't work, it could be that you're uploading only 1 side of your ID card, If that's the case take a picture of both sides , combine them into one file and then upload it again. 

In any case, I'll leave you a link of an old discussion that may be helpful: [Why can't I submit my application for Github Education Pack?](https://github.com/orgs/community/discussions/64963)

Remember that you must use your school's educational email to make the application and it must be linked to your GitHub account.

**Reply by LucasMontoya4** [reply]
Hi @davevad93, thanks a lot for the tips. I'm afraid the same is still happening, even though I have done all of the steps you suggested. The same happens on Firefox (safari was the original). I feel like it has something to do with the fact that the first application was rejected, because that seemed to go through and be processed quickly.

**Reply by davevad93** [reply]
Hi @LucasMontoya4 , you're welcome and I'm sorry that you haven't be able to solve it. I don't think it is related to your first application being rejected and I'm pretty sure of that because my application was rejected 2 times before being accepted (it was accepted the 3rd time). I don't know what could be the cause of your problem so I suggest you to contact [GitHub Education Support](https://support.github.com/request/education) directly. I hope you can solve it!

**Reply by LucasMontoya4** [reply]
Thank you! I'll get on that now.
